README.md

```diff
@@ -262,6 +262,22 @@ magic methods will be striginfied in JSON as simple objects) and you will not be
 features like mixed indent, pre/post render hooks but in this mode you will get exact same
 output as in pug-js.
 
+### Write locals object to json file with pugjs
```

Comment:
Hi, as this part comes with pugjs=>true, it should be a sub-section of the previous one, so I would mark it as a h4 (`####`)